### ISSUE TITLE
[FEATURE] Improve Pathfinder 2e support (spells, passive perception)

### ISSUE THREAD

**Issue by xanstin** [illandril/FoundryVTT-token-tooltips#37]
I am trying to get the effect like 5E where I have the star with the number 1 over it for a spell slot tracker in PF2

**Comment by illandril** [comment]
There isn't currently a way to do that for custom properties.
If you can give me the details on how to find spell slot details for PF2, I can build it in as a standard property. The only reason it's not there now is that I don't play PF2 and nobody who uses it has given me the details I'd need to find them reliably.

**Comment by xanstin** [comment]
Ok let me work on that. I thought I found it yesterday to only find out it didn’t match up with the sheet numbers.

**Comment by xanstin** [comment]
Perception - data.attributes.perception.value
Focus Points (Type of Spell slot pool) data.resources.focus

**Comment by illandril** [comment]
I took another look through the system, and I think I've found how to get everything, except for "Vancian" spellcasting (Prepared, non-Flexible) - I can't think of a good way to handle those in a tooltip.

It's also a bit odd for actors with multiple spellcasting entries that each use level-based spell slots - they'll have multiple "level 1 spell slots" show up in the tooltip with no way to distinguish between them (except that they'll be in the same order they would be in their character sheet).

I should have spell data and passive perception showing in a release by the end of the day Sunday (probably earlier, but it all depends on how much free time I can get to test things out to make sure I didn't break anything in the process).

![Screenshot from 2021-10-04 11-52-13](https://user-images.githubusercontent.com/469205/135884004-b647ee49-dd64-43d8-a566-6300a37d8a7c.png)

Are there any other custom fields you configure for PF2 that you think would be useful for everyone?
